monkey/infection_monkey/post_breach/actions/add_user.py

```diff
@@ -1,21 +1,61 @@
 import datetime
+
+from common.data.post_breach_consts import POST_BREACH_BACKDOOR_USER
 from infection_monkey.post_breach.pba import PBA
 from infection_monkey.config import WormConfiguration
 
 
-__author__ = 'danielg'
+class BackdoorUser(PBA):
+    def __init__(self):
+        linux_cmds, windows_cmds = BackdoorUser.get_commands_to_add_user(
+            WormConfiguration.user_to_add, WormConfiguration.remote_user_pass)
+        super(BackdoorUser, self).__init__(
+            POST_BREACH_BACKDOOR_USER,
+            linux_cmd=' '.join(linux_cmds),
+            windows_cmd=windows_cmds)
+
+    @staticmethod
+    def get_commands_to_add_user(username, password):
```

Comment:
I would move this function + `get_linux_commands_to_add_user, get_windows_commands_to_add_user, get_linux_commands_to_delete_user, get_windows_commands_to_delete_user` to a new separate file NOT UNDER post_breach

Reply:
889c8a23